src/components/NewGalaxyModal.js

```diff
@@ -68,9 +82,11 @@ export const NewGalaxyModal = _.flow(
     return onSuccess()
   })
 
-  const deleteButton = h(ButtonSecondary, { style: { marginRight: 'auto' }, onClick: () => setViewMode('deleteWarn') }, ['Delete'])
-
   const renderActionButton = () => {
```

Comment:
Changes below were upon the UX request to add disabled `Delete` and `Pause` buttons to transitionary status modals.